test/e2e/jmx/resources/sample_apps/kafka-deployment.yaml

```diff
@@ -0,0 +1,122 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: zookeeper-service
+spec:
+  selector:
+    app: zookeeper
+  ports:
+    - port: 2181
+      targetPort: 2181
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: zookeeper-deployment
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: zookeeper
+  template:
+    metadata:
+      labels:
+        app: zookeeper
+    spec:
+      containers:
+        - name: zookeeper
+          image: wurstmeister/zookeeper:latest
```

Comment:
Instead of using latest, can we pin a version so that it never changes?

Or maybe better yet -- we push an image to our own ECR repo?

Reply:
Good point, I'll add it to our own ECR repo.